test/stdpar/tests/reduce.cpp

```diff
@@ -15,8 +15,6 @@ int main()
 {
   std::vector<int> v(N);
   std::fill(std::execution::par_unseq, v.begin(), v.end(), 42);
-  int sum = std::reduce(std::execution::par_unseq, v.begin(), v.end(), 100, [](int a, int b) {
-    return a + b;
-  });
+  int sum = std::reduce(std::execution::par_unseq, v.begin(), v.end(), 100);
```

Comment:
Keep both of the `std::reduce` calls in the test program.  I expect that the call with the custom lambda would catch more bugs over time than the one with the default reduction operator.

The test program was very minimal because I wanted to get something quickly.  I expect the test program to grow over time.